DEVDOCS.md

```diff
@@ -0,0 +1,60 @@
+#Encore Link Developer Documentation
+EncoreLink is a web application currently built with (from back to front):
+
+* [PostgreSQL Database](https://www.postgresql.org/)
+* [Loopback API Framework](https://loopback.io/)
+* [Node.js Express Web Framework](http://expressjs.com/)
+* [Redux State Container](http://redux.js.org/)
+* [React.js UI Library](https://facebook.github.io/react/)
+* [ES6 JavaScript](http://es6-features.org) with [Babel JS compiler](https://babeljs.io/)
+* [Foundation by Zurb Front End Framework](http://foundation.zurb.com/)
+* [Webpack Module Bundler](https://webpack.github.io/)
+* [Jest Testing Framework](https://facebook.github.io/jest/)
+
+##Data
+We use Loopback to create and control our data. Once you install it with npm and get the shell database created as described on our [README](README.md), you can run a little interactive command line app that walks you through creating your data/API.
+
+* The Loopback site has a good [Getting Started](http://loopback.io/getting-started/) page that shows you the basics of using that command line tool.
+* The main files that represent the model in our code are the .json files in the `common\model` folder. Examples are `user.json` and `event.json`.
+* JSON is just data, so comments are not allowed. Plus Loopback generates these JSO files, so our comments would be toast if we altered the model using the Loopback CLI.
+ * NOTE: We might look to commenting these files in the future by either minifying to remove comments or adding a comment element
+
+##State
+We use Redux to control our application state.
+
+* We use Redux because it ensures your state transitions are understandable, because there is only one unique next state for any starting state/action combination
+* Check out [this page](http://redux.js.org/docs/introduction/index.html) to get up on Redux fundamentals, or watch [these videos](https://egghead.io/courses/getting-started-with-redux)
+* Of course, the devil is in the implementation. We use all sorts of funky advanced tricks in our implementation, which makes understanding how to code up a new state transition  tricky. See below for a walkthrough example. The basic file separation is:
+ * **Actions** are defined in `client\src\actions`. As of Nov 3, 2016, we have one file, `index.js` that represents our actions
+ * **Initial state** and **Reducers** are defined in `client\src\reducers`. As of Nov 3, 2016, we have two main reducers in EncoreLink - the `userManager` and the `eventManager`.
```

Comment:
Also not sure if we care to define the reducer files. It might be more useful to say that `Reducers are functions that define how client state is mutated`, and link to the redux docs on reducers.